src/main/scala/dxWDL/compiler/GenerateIRWorkflow.scala

```diff
@@ -612,6 +631,52 @@ case class GenerateIRWorkflow(wf : WorkflowDefinition,
          applet)
     }
 
+    private def addCustomReorgStage(wfName: String,
+                                    wfSourceStandAlone: String,
+                                    wfOutputs: Vector[(CVar, SArg)],
+                                    reorgAttributes: ReorgAttrs
+                                   ) : (IR.Stage, IR.Applet) = {
+
+        val appletKind = IR.AppletKindWorkflowCustomReorg(reorgAttributes.appId)
+
+        // will throw error if there is no status string. Should consider checking there i s only one.
+        val reorgStatusInput: (CVar, SArg) = wfOutputs.filter(x=> x._1.name == Utils.REORG_STATUS).head
+
+        val configFile: Option[WomSingleFile] = reorgAttributes.reorgConf match {
+            case "" => None
+            case x: String => Some(WomSingleFile(x))
+        }
+
+        val appInputs = Vector(
+            reorgStatusInput._1,
```

Comment:
Please try to avoid `reorgStatusInput._1`, it makes the code less readable.

Reply:
Okays will unpack.